var/spack/repos/builtin/packages/py-easybuild-easyconfigs/package.py

```diff
@@ -12,9 +12,16 @@ class PyEasybuildEasyconfigs(PythonPackage):
     """
 
     homepage = 'http://hpcugent.github.io/easybuild/'
-    url      = 'https://pypi.io/packages/source/e/easybuild-easyconfigs/easybuild-easyconfigs-3.1.2.tar.gz'
+    url      = 'https://pypi.io/packages/source/e/easybuild-easyconfigs/easybuild-easyconfigs-4.0.0.tar.gz'
+    maintainers = ['boegel']
 
+    version('4.0.0', sha256='90d4e8f8abb11e7ae2265745bbd1241cd69d02570e9b4530175c4b2e2aba754e')
     version('3.1.2', '13a4a97fe8a5b9a94f885661cf497d13')
 
-    depends_on('py-easybuild-framework@3.1:', when='@3.1:', type='run')
-    depends_on('py-easybuild-easyblocks@3.1.2:', when='@3.1.2', type='run')
+    depends_on('python@2.6:2.8', when='@:3', type=('build', 'run'))
+    depends_on('python@2.6:2.8,3.5:', when='@4:', type=('build', 'run'))
+
+    for v in ['@3.1:', '@4.0:']:
+        depends_on('py-easybuild-framework' + v, when=v, type='run')
```

Comment:
Same here, in theory `easybuild-easyconfigs` 4.0.1 could require `easybuild-framework` 4.0.1